apps/bridge-dapp/src/pages/Account/Transactions/index.tsx

```diff
@@ -0,0 +1,223 @@
+import { type FC, useState, useCallback } from 'react';
+import { Outlet } from 'react-router-dom';
+import { Typography } from '@webb-tools/webb-ui-components';
+import { DeleteBinIcon, UploadLine, Download } from '@webb-tools/icons';
+
+import ActionsDropdown from '../../../components/ActionsDropdown';
+import ClearTxHistoryModal from '../../../containers/ClearTxHistoryModal';
+import UploadTxHistoryModal from '../../../containers/UploadTxHistoryModal';
+import HiddenValueEye from '../../../components/HiddenValueEye';
+import NoTx from '../NoTx';
+import TxTableContainer from '../../../containers/TxTableContainer';
+import { TxTableItemType } from '../../../containers/TxTableContainer/types';
+import downloadTxHistory, {
+  type DownloadTxType,
+} from '../../../utils/downloadTxHistory';
+
+import { randNumber, randEthereumAddress } from '@ngneat/falso';
+const fakeTxData: TxTableItemType[] = [
```

Comment:
Maybe we can extract fake data into a reusable file? It seems we're rewriting this in more than one place?

Reply:
These will all be removed when integrating with the backend. Keeping the fake data in each file will make it easier for us to remove them later. If creating a new file for reusable fake data, we have to delete it eventually.